### PR TITLE
Adds 3.10 known issue to remove various components

### PR DESCRIPTION
**Pull request by stevsmit**

### CODE REVIEW

**Review by jonathankingfc — APPROVED** [review]
LGTM

File: modules/operator-upgrade.adoc [hunk 1]
```diff
@@ -43,6 +43,133 @@ To update {productname} from one minor version to the next, for example, 3.9 ->
...
+----
+
+. Remove the `quayregistry-quay-config-editor` route object by entering the following command:
++
+[source,terminal]
+----
+$ oc delete route quayregistry-quay-config-editor
+----
+
+. Obtain the `quay-quay-config-editor` deployment object by entering the following command:
```
Comment (SeanZhao-redhat) [inline modules/operator-upgrade.adoc:85]: to be consistent, pls change all "quay-quay-xxx" to "quayregistry-quay-xxx" (quite a few in this section).  I worked on 2 env, which has different quay registry name, to be a formal doc, suggest to use only one, "quayregistry-quay-xxx" is more common.

File: modules/operator-upgrade.adoc [hunk 2]
```diff
@@ -43,6 +43,133 @@ To update {productname} from one minor version to the next, for example, 3.9 ->
...
+---
+quay-quay-config-editor
+---
+----
+
+. Remove the `quay-quay-config-editor` deployment object by entering the following command:
++
+[source,terminal]
+----
+$ oc delete route quayregistry-quay-config-editor
```
Comment (SeanZhao-redhat) [inline modules/operator-upgrade.adoc:105]: typo, here "route" should be "deployment", -> oc delete deployment quayregistry-quay-config-editor

File: modules/operator-upgrade.adoc [hunk 3]
```diff
@@ -43,6 +43,133 @@ To update {productname} from one minor version to the next, for example, 3.9 ->
...
+----
+
+. Obtain the `quay-quay-config-editor` service object by entering the following command:
++
+[source,terminal]
+----
+$ $ oc get svc | grep config
```
Comment (SeanZhao-redhat) [inline modules/operator-upgrade.adoc:112]: typo, double "$" signal

File: modules/operator-upgrade.adoc [hunk 4]
```diff
@@ -43,6 +43,133 @@ To update {productname} from one minor version to the next, for example, 3.9 ->
...
+[source,terminal]
+----
+$ oc delete service quay-quay-config-editor
+----
+
+. Obtain the `quayregistry-quay-config-editor-credentials` secret by entering the following command:
++
+[source,terminal]
+----
+$ oc get secret | grep editor
```
Comment (SeanZhao-redhat) [inline modules/operator-upgrade.adoc:133]: suggest to "oc get secret | grep config-editor" to be consistent with above

**Pull request merged**